### ISSUE TITLE
Error `path.replace is not a function`

### ISSUE THREAD

**Issue by dima74** [nuxt/webpackbar#43]
Please consider casting `path` in this line to string:

https://github.com/nuxt/webpackbar/blob/73ef3ae64f92a049500d5494f87b684024926d8f/src/utils/index.js#L42

Because in my experience `path` can be number.

---

I don't have any good reproduction (sorry), so my proposal is to just add cast to string. Just to be complete, here is full stack trace with this error:

```
PATH/node_modules/@nuxt/webpack/node_modules/webpackbar/dist/utils/index.js:58
  return path.replace(cwd, '');
              ^

TypeError: path.replace is not a function
    at shortenPath (PATH/node_modules/@nuxt/webpack/node_modules/webpackbar/dist/utils/index.js:58:15)
    at PATH/node_modules/@nuxt/webpack/node_modules/webpackbar/dist/plugin.js:182:43
    at SyncHook.eval [as call] (eval at create (PATH/node_modules/@nuxt/webpack/node_modules/webpack/node_modules/tapable/lib/HookCodeFactory.js:19:10), <anonymous>:7:1)
    at SyncHook.lazyCompileHook (PATH/node_modules/@nuxt/webpack/node_modules/webpack/node_modules/tapable/lib/Hook.js:154:20)
    at Watchpack.watcher.compiler.watchFileSystem.watch (PATH/node_modules/@nuxt/webpack/node_modules/webpack/lib/Watching.js:139:33)
    at Object.onceWrapper (events.js:285:13)
    at Watchpack.emit (events.js:197:13)
    at Watchpack.EventEmitter.emit (domain.js:446:20)
    at Watchpack._onChange (PATH/node_modules/watchpack/lib/watchpack.js:118:7)
    at Watchpack.<anonymous> (PATH/node_modules/watchpack/lib/watchpack.js:109:8)
    at Watcher.emit (events.js:197:13)
    at Watcher.EventEmitter.emit (domain.js:446:20)
    at PATH/node_modules/watchpack/lib/DirectoryWatcher.js:101:9
    at Array.forEach (<anonymous>)
    at DirectoryWatcher.setFileTime (PATH/node_modules/watchpack/lib/DirectoryWatcher.js:99:42)
    at DirectoryWatcher.onFileAdded (PATH/node_modules/watchpack/lib/DirectoryWatcher.js:250:7)
    at FSWatcher.emit (events.js:197:13)
    at FSWatcher.EventEmitter.emit (domain.js:446:20)
    at FSWatcher.<anonymous> (PATH/node_modules/watchpack/node_modules/chokidar/index.js:199:15)
    at FSWatcher._emit (PATH/node_modules/watchpack/node_modules/chokidar/index.js:241:5)
    at FSWatcher.<anonymous> (PATH/node_modules/watchpack/node_modules/chokidar/lib/nodefs-handler.js:465:12)
    at FSReqCallback.oncomplete (fs.js:159:5)

```

<div align="right"><sub><em>This question is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/webpackbar/issues/c34">#c34</a>)</em></sub></div>

**Comment by nfour** [comment]
This happens when you `--watch`. Still an issue.

**Comment by mohsinulhaq** [comment]
any update here?

**Comment by pi0** [comment]
fixed with #45
